Library/Formula/ne.rb

```diff
@@ -0,0 +1,19 @@
+class Ne < Formula
+  desc "The nice editor"
+  homepage "http://ne.di.unimi.it"
+  url "http://ne.di.unimi.it/ne-3.0.1.tar.gz"
+  sha256 "92b646dd2ba64052e62deaa4239373821050a03e1b7d09d203ce04f2adfbd0e4"
+
```

Comment:
Oh, building from the repository requires LaTeX for the documentation. Nevermind, then.